cluster.go

```diff
@@ -156,10 +156,15 @@ func (c *Cluster) setupConsensus() error {
 		startPeers = peersFromMultiaddrs(c.config.Bootstrap)
 	}
 
+	dataFolder := c.config.ConsensusDataFolder
```

Comment:
Yeah, the reason I did not do it when loading the config, is that if you set ConsensusDataFolder there (or anywhere else) to a expected value  when it's empty, then you need to remove it when you save the configuration (configuration is saved sometimes, for example when peers are added) and make sure it's exactly as it was loaded. So I went the simpler way.

This way configuration stays the same.